.travis.yml

```diff
@@ -1,8 +1,3 @@
 language: node_js
 node_js:
-    - 4
-script: gulp
-before_script:
-    - npm install -g istanbul
-after_success:
-    - gulp coverage coveralls
```

Comment:
This would not work. 'gulp coverage' is enough since you dont't have any task called 'coverage coveralls'